### PR TITLE
MAGN-10225 disable the migration of 0.6.3 node. (Removal 6)

### PR DESCRIPTION
**Pull request by rk-lakshmanan**
### Purpose
References: [MAGN-10225](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-10225)
This PR is for the removal of code related to migration of versions below 0.7.1

### Declarations

Check these if you believe they are true

- [ ] The code base is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.

### Reviewers

@sharadkjaiswal

### CODE REVIEW

File: src/Migrations/RevitNodes/SunPath.cs [hunk 1]
```diff
@@ -1,91 +0,0 @@
...
-            migrationData.AppendNode(vectorAsPoint);
-
-            // Update connectors
-            migrationData.CreateConnector(sunPathNode, 0, vectorAsPoint, 0);
-
-            return migrationData;
-        }
-
-        [NodeMigration(from: "0.7.0.0", to: "0.7.3.0")]
-        public static NodeMigrationData Migrate_0700_to_0730(NodeMigrationData data)
```
Comment (sharadkjaiswal) [inline src/Migrations/RevitNodes/SunPath.cs]: We can't remove this one.

Comment (sharadkjaiswal) [inline src/Migrations/RevitNodes/SunPath.cs]: This node migration is for 0.7.0 to 0.7.3, what happens to 0.7.1 node, will that also execute this code?

Comment (rk-lakshmanan) [inline src/Migrations/RevitNodes/SunPath.cs]: It does not seem to hit this code.

Comment (sharadkjaiswal) [inline src/Migrations/RevitNodes/SunPath.cs]: For now let's leave such methods, don't remove them. I'll take a look later on.

**Pull request merged**